recipe/meta.yaml

```diff
@@ -1,46 +1,90 @@
 {% set name = "splink" %}
-{% set version = "3.6.0" %}
+{% set version = "3.9.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6bbef574b12ebf74e112960e2bc5d8baf3ca60f5d889a581dafb2f29173e90a8
+  sha256: 891996d3eaa6ff58cc3e3cd97c0aa0c45246a8b1c858cd4fe77f617e190f20ee
 
 build:
-  noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<38 or s390x]
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
+# the requirements refer to the duckdb-less installation documented at
+# https://moj-analytical-services.github.io/splink/getting_started.html#duckdb-less-installation
 requirements:
   host:
-    - python >=3.7,<4.0
+    - python
     - pip
-    - poetry >=0.12
+    - setuptools
+    - wheel
+    - poetry
```

Comment:
```suggestion
    - poetry-core >=1.0.8
```
https://github.com/moj-analytical-services/splink/blob/v3.9.10/pyproject.toml#L84
Build backend is poetry-core, setuptools and wheel are not required.